R/tm_a_pca.R

```diff
@@ -13,16 +13,46 @@
 #' It controls the font size for plot titles, axis labels, and legends.
 #' - If vector of `length == 1` then the font sizes will have a fixed size.
 #' - while vector of `value`, `min`, and `max` allows dynamic adjustment.
-#' @templateVar ggnames "Elbow plot", "Circle plot", "Biplot", "Eigenvector plot"
-#' @template ggplot2_args_multi
+#' @param ggplot2_args `r roxygen_ggplot2_args_param("Elbow plot", "Circle plot", "Biplot", "Eigenvector plot")`
+#' @param decorators `r roxygen_decorators_param("tm_a_pca")`
 #'
 #' @inherit shared_params return
 #'
+#' @section Decorating `tm_a_pca`:
+#'
+#' This module generates the following objects, which can be modified in place using decorators:
+#' - `elbow_plot` (`ggplot2`)
+#' - `circle_plot` (`ggplot2`)
+#' - `biplot` (`ggplot2`)
```

Comment:
is it `biplot` or `biplot_plot`?
Example breaks on BIPLOT

<img width="879" alt="image" src="https://github.com/user-attachments/assets/b9c4f51b-e6ad-41c5-9ea9-41e9722acbe6">

Reply:
In the example we use `biplot_plot` in the decorator.
`biplot = caption_decorator("I am a PCA / bipot", "biplot_plot"),`


So I think this is an error in the example, that we should change to 

`biplot = caption_decorator("I am a PCA / bipot", "biplot"),`